html-forms/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>html-forms</title>
+</head>
+
+<body>
+  <h1>Coding Form!</h1>
+  <div class="form-example">
+    <label for="email">Email</label>
+    <div><input type="email" name="email" id="email" required></div>
+  </div>
+<form>
+  <div>Why is coding cool?</div>
+  <select name="option">
+  <option value="select an option">Select an option</option>
+  <option value="because it's intresting">Because it's intresting</option>
+  <option value="because it's fun">Because it's fun</option>
+  <option value="because it's powerful">Because it's powerful</option>
+  </select>
+</form>
+<form>
+  <div>Favorite Language (choose one)</div>
+  <input type="radio" name="language" value="html"/>
+  HTML
+  <input type="radio" name="language" value="css" />
+  CSS
+  <input type="radio" name="language" value="javascript" />
+  JavaScript
+</form>
+<form>
+  <div>Would you like to sign up for more information?</div>
+  <input type="checkbox" name="service"
+  value="sign me up" /> Sign me up!
+</form>
+<form>
+<div>Comments</div>
+ <textarea name="comments" cols="30" rows="10"></textarea>
+</form>
```

Comment:
you are missing the `input` with the type of `submit` so the form can be submitted.